src/main/java/edu/harvard/iq/dataverse/api/Access.java

```diff
@@ -130,9 +130,19 @@
 import jakarta.ws.rs.core.MediaType;
 import static jakarta.ws.rs.core.Response.Status.FORBIDDEN;
 import static jakarta.ws.rs.core.Response.Status.UNAUTHORIZED;
+
+import org.eclipse.microprofile.openapi.annotations.Operation;
+import org.eclipse.microprofile.openapi.annotations.media.Content;
+import org.eclipse.microprofile.openapi.annotations.media.Schema;
+import org.eclipse.microprofile.openapi.annotations.parameters.RequestBody;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponse;
+import org.eclipse.microprofile.openapi.annotations.responses.APIResponses;
+import org.eclipse.microprofile.openapi.annotations.tags.Tag;
 import org.glassfish.jersey.media.multipart.FormDataBodyPart;
 import org.glassfish.jersey.media.multipart.FormDataParam;
 
+import com.mashape.unirest.request.body.MultipartBody;
```

Comment:
I'm the one who added Unirest but it's one of the libraries I'm hoping we can remove some day in favor of using the Jakarta EE standard. Do we need it here? I see a possible solution at https://stackoverflow.com/questions/64079847/how-to-provide-swagger-annotation-for-multipartformdatainput-in-resteasy-with-qu/65339649#65339649 . Is it worth trying it here and other places in this PR where Unirest is being used?